ui/apps/platform/src/css/trumps.css

```diff
@@ -80,6 +80,11 @@ button.pf-c-toggle-group__button:disabled {
     border-width: 1px !important;
 }
 
+/* Un-override the above override when the <Select> component is disabled */
+.pf-c-select__toggle.pf-m-disabled::before {
```

Comment:
Before this change:
<img width="785" alt="image" src="https://user-images.githubusercontent.com/1292638/199264013-406396eb-a760-4b3b-8c08-b29abc72f5ac.png">

After this change:
<img width="785" alt="image" src="https://user-images.githubusercontent.com/1292638/199264086-3b8c7623-3319-4130-91bc-c46177ada1b7.png">

Disabled `<Select>` styles on the PatternFly site:
<img width="940" alt="image" src="https://user-images.githubusercontent.com/1292638/199264210-bf862de9-f68a-429c-8a2a-e3ef85e80aa8.png">

Reply:
Also note the subtle color difference between the top and bottom dropdowns. This is due to an `opacity: 0.5;` rule from the `ui-components` package that is applying to `<button>` elements. The top `<Select>` uses a button as the main element while the bottom uses a `<div>` due to the difference PF options in use.